proposals/2021/Add AuditFilingStatus.md

```diff
@@ -0,0 +1,52 @@
+# Add AuditFilingStatus
+
+## Overview
+
+This proposal is to add the `AuditFilingStatus` element as a child of `ReportType` element, and to restrict the options with `Initial filing` and `Amended filing`.
```

Comment:
Is it under `ReportType` element or `Report` element instead? @corymosiman12

Reply:
![image](https://user-images.githubusercontent.com/28991233/112071634-9513b380-8b46-11eb-8f18-1ef718bce08d.png)

@JieXiong9119 you would end up putting it under the 'ReportType' element, which gets implemented in the 'Report' element. Make sense?

Reply:
Which you have already done, my b. You are good.